Jenkinsfile

```diff
@@ -229,76 +229,88 @@ pipeline {
                     }
                 }
 
-                /*stage('gfx908 GCC Debug') {
-                    agent{ label rocmnode("gfx908") }
-                    steps{
-                        buildJob('g++-5', '-DMIOPEN_TEST_GFX908=On -DBUILD_DEV=On -DCMAKE_BUILD_TYPE=debug', "", image, "")
-                    }
-                }*/
```

Comment:
Why is this commented it out?

Reply:
Dunno it was commented out at some point. I just deleted it.